packages/openid4vc/src/openid4vc-issuer/router/jwtIssuerVcMetadataEndpoint.ts

```diff
@@ -0,0 +1,31 @@
+import type { SdJwtVcIssuerMetadata } from '@credo-ts/core'
+import type { Response, Router } from 'express'
+import { getRequestContext, sendJsonResponse, sendUnknownServerErrorResponse } from '../../shared/router'
+import { OpenId4VcIssuerService } from '../OpenId4VcIssuerService'
+import type { OpenId4VcIssuanceRequest } from './requestContext'
+
+export function configureJwtVcIssuerMetadataEndpoint(router: Router) {
+  router.get('/.well-known/jwt-vc-issuer', async (_request: OpenId4VcIssuanceRequest, response: Response, next) => {
```

Comment:
This won't be hosted at the correct path looking at the spec. It will be hosted at `<url>/oid4vci/<uuid>/.well-known/jwt-vc-issuer`, while it should be hosted at `<url>/.well-known/jwt-vc-issuer/oid4vci/<uuid>`

We have this issue for other documents as well: https://github.com/openwallet-foundation/credo-ts/issues/2221